lib/ApiRequestor.php

```diff
@@ -117,6 +117,31 @@ public function handleApiError($rbody, $rcode, $rheaders, $resp)
         }
     }
 
+    private static function _defaultHeaders($apiKey)
+    {
+        $langVersion = phpversion();
+        $uname = php_uname();
+        $appInfo = Stripe::getAppInfoAsString();
+        $uaString = 'Stripe/v1 PhpBindings/' . Stripe::VERSION;
+        $ua = array(
+            'bindings_version' => Stripe::VERSION,
+            'lang' => 'php',
+            'lang_version' => $langVersion,
+            'publisher' => 'stripe',
+            'uname' => $uname,
+        );
+        if ($appInfo !== null) {
+            $uaString .= ' ' . $appInfo;
+            $ua['application'] = $appInfo;
```

Comment:
Is it possible to do nested values in here - i.e., leave the 'object' as an object?  It might be easier to grok [0] for analytics etc. if it didn't require parsing the string.

[0] _This might actually be the first time I've typed `grok` on the internet._